docs/src/rsh/errors/index.md

```diff
@@ -2106,11 +2106,32 @@ can either abstract the arguments into a new `{!rsh} Data` type or make separate
 
 ## {#RE0109} RE0109
 
```

Comment:
@EBEREGIT We need to add a note under the heading:

`This error will not happen after version v0.1.`

See an example at [RE0047](https://docs.reach.sh/rsh/errors/#RE0047)